config/jobs/recipe-core-main.yml

```diff
@@ -3,7 +3,7 @@ jobs:
     - php: 7.1
       env:
         - DB=MYSQL
-        - REQUIRE_RECIPE_CORE=4.5.x-dev
```

Comment:
should it be `REQUIRE_INSTALLER`?

Reply:
nope, should be REQUIRE_RECIPE_CORE

Reply:
haha, I think I reviewed a different thing there..